### ISSUE TITLE
random key search

### ISSUE THREAD

**Issue by fernand77** [brichard19/BitCrack#185]
Please created functions random search keys hex please!

**Comment by TheDailySpank** [comment]
Currently, it's already implemented in this fork: https://github.com/pikachunakapika/BitCrack

**Comment by fernand77** [comment]
No, that version is not working, it is not for nothing that the author sent it to the archive. it lays out points only in the 256 range

**Comment by TheDailySpank** [comment]
When I tested it, it appeared to create 33,000,000 randomly selected start points and then started incrementing each one of those. Sure, it's not truly random in that sense, but how would you go about generating 125,000,000 random numbers a second?

If you're referring to the "bit range", I just ran it and saw 252, 254, and 256 listed.

**Comment by fernand77** [comment]
yes, bit range. he jumps from 252 to 256 only

**Comment by Coinpiet** [comment]
when you use random funktion  -r with --share then it works 

so between the shares it will create all random starting points in the given search space.

`-08-25.13:59:57] [Info] Share 90 of 120
[2019-08-25.13:59:57] [Info] Compression: compressed
[2019-08-25.13:59:57] [Info] Starting at: 0000000000000000000000000000000000000000000000003D0624DD2F1A9F72
[2019-08-25.13:59:57] [Info] Ending at:   0000000000000000000000000000000000000000000000003D1EB851EB851E6A
[2019-08-25.13:59:57] [Info] Generating random starting points
[2019-08-25.13:59:57] [Info] Counting by: 0000000000000000000000000000000000000000000000000000000000000001
[2019-08-25.13:59:57] [Info] Initializing GeForce GTX 1070
[2019-08-25.13:59:57] [Info] Generating 24,576,000 starting points (937.5MB)
[2019-08-25.13:59:57] [Info] Starting point sample: 0000000000000000000000000000000000000000000000003D067C0C08D9913A (62 bit range)
[2019-08-25.13:59:57] [Info] Starting point sample: 0000000000000000000000000000000000000000000000003D0E86349A31F8A6 (62 bit range)
[2019-08-25.13:59:57] [Info] Starting point sample: 0000000000000000000000000000000000000000000000003D14CE227B0C1BFD (62 bit range)`
